docs/feature-gates.md

```diff
@@ -43,6 +43,7 @@ example, to enable `AntreaProxy` on Linux, edit the Agent configuration in the
 | `NetworkPolicyStats`    | Agent + Controller | `true`  | Beta  | v0.10         | v1.2         | N/A        | No                 |       |
 | `NodePortLocal`         | Agent              | `false` | Alpha | v0.13         | N/A          | N/A        | Yes                |       |
 | `Egress`                | Agent + Controller | `false` | Alpha | v1.0          | N/A          | N/A        | Yes                |       |
+| `NodeIPAMController`    | Controller         | `false` | Alpha | v1.4          | N/A          | N/A        | Yes                |       |
```

Comment:
I thought we were trying to merge this in time for v1.3, bit v1.4 works for me since I'm late with my review :)

cc @vicky-liu

Reply:
We might need a pre-release build for the uses case the will need to leverage this feature